src/Repository/MpxMediaType.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\media_mpx\Repository;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\media\Entity\MediaType;
+use Drupal\media_mpx\Exception\MediaTypeDoesNotExistException;
+use Drupal\media_mpx\Exception\MediaTypeNotAssociatedWithMpxException;
+use Drupal\media_mpx\Plugin\media\Source\Media;
+
+/**
+ * Class MpxMediaType.
+ *
+ * @package Drupal\media_mpx\Repository
+ */
+class MpxMediaType {
```

Comment:
👍 - i bet there's lots of other places in the code we could reuse this.

Reply:
Totally. I'll keep an eye in the upcoming tasks for further consolidation.